client/my-sites/themes/index.web.js

```diff
@@ -23,8 +23,11 @@ export default function( router ) {
 		if ( isLoggedIn ) {
 			router( '/design/:tier(free|premium)?', multiSite, navigation, siteSelection );
 			router( '/design/:tier(free|premium)?/:site_id', singleSite, navigation, siteSelection );
+			router( '/design/:tier(free|premium)?/filter/:filter', multiSite, navigation, siteSelection );
+			router( '/design/:tier(free|premium)?/filter/:filter/:site_id', singleSite, navigation, siteSelection );
```

Comment:
Hrm, I guess we can't whitelist all combinations if we're going to concatenate them with `+`es and not even normalize that string :-/